addons/udes_stock/models/stock_picking.py

```diff
@@ -105,6 +105,14 @@ class StockPicking(models.Model):
     # override batch_id to be copied
     batch_id = fields.Many2one("stock.picking.batch", copy=True)
 
+    u_batch_user_id = fields.Many2one(
+        "res.users",
+        "Batch User",
+        compute="_compute_batch_user",
+        store=False,
+        help="User responsible for the batch",
+    )
+
```

Comment:
We can make use of the `related` parameter to avoid having to compute the field e.g. 
```
u_batch_user_id = fields.Many2one(related="batch_id.user_id", string=BLAH ETC)
```